Sources/Foundation/FileManager+POSIX.swift

```diff
@@ -741,32 +741,38 @@ extension FileManager {
             if rmdir(fsRep) == 0 {
                 return
             } else if errno == ENOTEMPTY {
+                #if os(Android)
+                let ps = UnsafeMutablePointer<UnsafeMutablePointer<Int8>>.allocate(capacity: 2)
```

Comment:
Do we really need to duplicate this or can we get away with a rebinding of the memory?

Reply:
No longer needed after your swiftlang/swift#74829, so removed in #5010.